components/FlexibleForms/TextField.spec.tsx

```diff
@@ -83,4 +83,34 @@ describe('TextField', () => {
     );
     expect(screen.getByText('Example error'));
   });
+
+  it('should render a textarea when the as prop is specified', () => {
```

Comment:
Could we have the same test but to show that input is rendered when the as prop isn't provided

Reply:
👍 done